packages/trader/src/AppV2/Components/TradeTypeList/trade-type-list.tsx

```diff
@@ -52,22 +48,6 @@ const TradeTypeList: React.FC<TTradeTypeListProps> = ({
                         'trade-type-list-category__border': category.items && category.items.length > 0,
                     })}
                 >
-                    <div className='trade-type-list-category-header'>
```

Comment:
If this part of code is not used anywhere else, should we clean the css (for trade-type-list-category-header, trade-type-list-category-header-title,trade-type-list-category-header-button)?